src/Resources/ContactProperties.php

```diff
@@ -51,7 +51,7 @@ public function get($name)
      */
     public function create(array $property)
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/properties';
+        $endpoint = "https://api.hubapi.com/contacts/v2/properties";
```

Comment:
Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)